lua/tests/infoview/pin_spec.lua

```diff
@@ -278,34 +283,44 @@ describe('infoview pins', helpers.clean_buffer('lean', dedent[[
         h37 : p
         ⊢ q ∨ p
       ]], vim.api.nvim_buf_get_name(0)))
```

Comment:
(this was a duplicate assertion I believe)

Reply:
In the original code it's meant to be a "guard assertion" -- meaning it was here so that if someone introduces a test before or after this one and doesn't realize tests are "dirty" and leak state from one to the other, this one should now fail in a more transparent way (by saying "the initial contents of the infoview are different than expected"). But let me know if you still think it looks duplicated, I didn't fully think about how useful the guard is here.